src/renderer/components/shared/SharedUtils.js

```diff
@@ -1,21 +1,33 @@
 export const META_CONCEPTS = new Set(['entity', 'relation', 'attribute', 'role']);
 
+export const interfaceTypes = {
+  SCHEMA_DESIGNER: 'SCHEMA_DESIGNER',
+  VISUALISER: 'VISUALISER',
+};
+
+export function getEdgeDefaultOptions() {
```

Comment:
this can just be an object, is there a specific reason why it is a function?

Reply:
no particular reason, will change it